mobile/src/main/java/ch/epfl/sdp/mobile/ui/tournaments/TournamentDetails.kt

```diff
@@ -507,3 +510,14 @@ private fun DetailsMatch(
     }
   }
 }
+
+/**
+ * A variation of [PagerState.currentPage] which displays the page which is the "most visible" on
```

Comment:
[Question] By "most visible" do you mean the one in the frontmost view?

Reply:
The one whose visible-width to total-width ratio is the highest.